server/controllers/authController.ts

```diff
@@ -76,3 +78,29 @@ export const getToken = catchAsync(
     res.status(200).end();
   },
 );
+
+export const getApplicationLogo = catchAsync(async (req: Request, res: Response) => {
```

Comment:
```suggestion
export const getOrganizationLogo = catchAsync(async (req: Request, res: Response) => {
```

Reply:
It is not actually organization logo but application logo. That is why it is called like this. In our case it doesn't make any difference but in every place in Microsoft docs it is called application logo

Reply:
application logo = 
![baseline_emoji_events_black_18dp](https://user-images.githubusercontent.com/24721386/111605335-7f189280-87d6-11eb-98ea-88e6052a09b9.png)

organization logo is dynamic. So these are two different things. I think it should be `getOrganizationLogo`

Reply:
You are mixing two things. Our application is one thing. The other one is application in Azure AD. This is the one I am talking about.

Reply:
yep 'Our application is one thing'. From our side Azure AD is the organization's application. So why did you name this function as `getApplicationLogo` when it's should name `getAzureADApplicationLogo`. I recommend simplifying this to `getOrganizationLogo` because `AzureADApplication === organization`

Reply:
No, you still don't understand. Don't say something like this : `AzureADApplication === organization` when you don't know. It is not true. AzureADApplication is one thing. AzureAD Organization is something different. I can call this function `getAzureADApplicationLogo`

Reply:
just something else than getApplicationLogo when it indicates ours logo, not organization's logo